src/plot_api/plot_schema.js

```diff
@@ -259,16 +259,16 @@ exports.getTraceValObject = function(trace, parts) {
         moduleAttrs = _module.attributes;
         valObject = moduleAttrs && moduleAttrs[head];
 
-        // then look in the global attributes
-        if(!valObject) valObject = baseAttributes[head];
-
-        // finally look in the subplot attributes
+        // then look in the subplot attributes
         if(!valObject) {
             var subplotModule = _module.basePlotModule;
             if(subplotModule && subplotModule.attributes) {
                 valObject = subplotModule.attributes[head];
             }
         }
+
+        // finally look in the global attributes
+        if(!valObject) valObject = baseAttributes[head];
```

Comment:
Why do we have look for base attributes last now?

Reply:
> Why do we have look for base attributes last now?

In case a specific trace alters the base behavior - which is assumed to require a longer redraw path, like [`box.name`](https://github.com/plotly/plotly.js/pull/1999/files#diff-bbaff551e896584401309e06c8e48a83R54) needing calc because it might be used as the coordinate.

Reply:
Makes sense, thanks!

This means though that the order by which the attributes are displayed on https://plot.ly/javascript/reference/ and friends will change. I'm not sure if this is a big deal cc @cldougl

Reply:
that's no problem (hoping to have search implemented on this page pretty soon anyway)

Reply:
Oh sorry. False alarm. This patch is in `getTraceValObject` which isn't called during `PlotSchema.get()`.